app/store/models/Registration.d.ts

```diff
@@ -82,10 +82,13 @@ export type DetailedRegistration = Pick<
   user: DetailedUser;
 };
 
-export type UnknownRegistration =
+export type UnknownRegistration = (
   | AnonymizedRegistration
   | PublicRegistration
   | ReadRegistration
   | SearchRegistration
   | PaymentRegistration
-  | DetailedRegistration;
+  | DetailedRegistration
+) & {
+  fetching?: boolean;
+};
```

Comment:
Yeah, haha, for some reason we keep track of whenever a specific registration is updating. Registration-objects don't contain `fetching`, so we need to add it as an extra property in the redux store to avoid type-errors.

Reply:
hmm okay is that even used?

Reply:
Yea, I think it might be used while registrering/unregistering from events